TrashMobMobile/ViewModels/CreateEventViewModelNew.cs

```diff
@@ -182,6 +184,18 @@ private void ValidateCurrentStep(object sender, PropertyChangedEventArgs e)
                     IsStepValid = false;
                 }
 
+                break;
+            //Step 2 validation 
+            case 1:
+                if (!string.IsNullOrEmpty(EventViewModel.Address.StreetAddress))
```

Comment:
It's actually common to have the street address be empty (most large parks don't have addresses on the map). Best to use city instead

Reply:
done